main.py

```diff
@@ -61,6 +61,9 @@ def __call__ (self, parser, namespace, values, option_string = None):
     parser.add_argument('--latent_iter_ratio', type=float, default=0.2, help="training iters that only use albedo shading")
     parser.add_argument('--albedo_iter_ratio', type=float, default=0, help="training iters that only use albedo shading")
     parser.add_argument('--jitter_pose', action='store_true', help="add jitters to the randomly sampled camera poses")
+    parser.add_argument('--jitter_center', type=float, default=0.2, help="amount of jitter to add to sampled camera pose's center (camera location)")
```

Comment:
Let's make the default 0.015? Also, set the defaults for the rest of them

Reply:
Let's see if @ashawkey knows of a counter-argument...

Reply:
I think these hyper-parameters can only be decided by running some experiments, do you have any results that shows 0.2 work well?

Reply:
0.2 was your default. ;-) In the Anya case study I included screenshots with different jitters: https://github.com/ashawkey/stable-dreamfusion/issues/263

Reply:
There's no rush to choose a different jitter default, we can do it later...